proto/lavanet/lava/dualstaking/delegator_reward.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package lavanet.lava.dualstaking;
+
+option go_package = "github.com/lavanet/lava/x/dualstaking/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+message DelegatorReward {
+    string index = 1; // index = provider + chainID + delegator
```

Comment:
I agree, but I've scaffolded this map like ignite would. Since I didn't want to deviate from the Cosmos scaffolded maps standard in our code (see EpochPayments, ProviderPaymentStorage, StakeStorage and more), I implemented this struct with an "index" field.

Should I change it?

Reply:
instead of keeping index, I'm keeping its components. See https://github.com/lavanet/lava/commit/59a09b37fc751c2632666683bcd7d068adae645d

Reply:
do we need the all of them, we have them when we fetch

Reply:
we need them to keep the delegator rewards data in the genesis state